### PR TITLE
Add workshops page

### PR DESCRIPTION
**Pull request by marcoow**
This adds the workshops page that lists all workshops from @mike-north's website. We're using a script to generate the markdown files with the content based on data from mike's site and then use these files in our normal jekyll build to create actual pages.

Design resources are here: 
[simplabs_workshop_website.zip](https://github.com/simplabs/simplabs.github.io/files/744346/simplabs_workshop_website.zip)

### CODE REVIEW

**Comment by ganeshbsub** [comment]
@marcoow Please review. I'll verify font sizes once I get home and have access to Photoshop.

File: _includes/head.html [hunk 1]
```diff
@@ -11,7 +11,9 @@
   <link rel="apple-touch-icon" sizes="72x72" href="/images/apple-72.png">
   <link rel="apple-touch-icon" sizes="114x114" href="/images/apple-114.png">
   <link rel="apple-touch-icon" sizes="144x144" href="/images/apple-144.png">
-  <link rel="stylesheet" href="//netdna.bootstrapcdn.com/bootstrap/3.1.1/css/bootstrap.min.css">
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.1.1/jquery.min.js"></script>
+  <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js" integrity="sha384-Tc5IQib027qvyjSMfHjOMaLkfuWVxZxUPnCJA7l2mCWNIpG9mGCD8wGNIcPD7Txa" crossorigin="anonymous"></script>
```
Comment (marcoow) [inline _includes/head.html]: I think we can maybe do without the Bootstrap JS library (and even jQuery). Actually it should be sufficient to toggle a class and define some CSS transitions for that class. That can probably be done with vanilla JS.

File: _layouts/workshop.html [hunk 2]
```diff
@@ -37,7 +37,10 @@
                         {{stage.title}}
                       </h4>
                       {{stage.description | markdownify}}
-                      <a class="custom-button" role="button" data-toggle="collapse" href="#agenda-items-{{forloop.index}}" aria-expanded="false" aria-controls="agenda-{{forloop.index}}">Agenda</a>
+
+                      {% unless stage.agenda_items == empty %}
```
Comment (marcoow) [inline _layouts/workshop.html]: I wonder if this is a bug in the data?

Comment (ganeshbsub) [reply _layouts/workshop.html]: I'm not sure but some agenda items seem to be empty arrays. That's why I added that bit of code.

**Comment by marcoow** [comment]
Can you check which ones are empty arrays and see whether that's the case in the data we get from Mike's API as well? Otherwise it's a but in the updater script. This seems pretty suspicious to me actually.

**Comment by ganeshbsub** [comment]
Fonts:
Main description: 18px
Meta: 12px
Module heading: 30px
Description: 16px

Start time/Agenda Heading: 24 px
Description: 16px

**Comment by marcoow** [comment]
@ganeshbsub: let's not use the exact font sizes from the PSD - more important font sizes are consistent with the rest of the page (which I think they are already).

**Comment by ganeshbsub** [comment]
Ah, okay. Cool.

**Comment by ganeshbsub** [comment]
@marcoow Ok, so I checked out the [data](https://api.mike.works/api/v1/courses/13/stages) from the API. Seems there are no Agenda items after all for one of the workshops.

**Comment by marcoow** [comment]
Which one is that? We should maybe ignore that one.

**Comment by ganeshbsub** [comment]
**Progressive Web Fundamentals**

Should I just remove the data or should we alter the script to ignore this one specifically?

**Comment by ganeshbsub** [comment]
On [Mike's website](https://mike.works/course/progressive-web-fundamentals-0d74af5), it says this is a **_New_** workshop

**Comment by marcoow** [comment]
Let's leave the check in for now - we can always manually delete generated markdown files for workshops that we don't want to have on the site anyway.

File: workshops/index.html [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+  </div>
+</div>
+
+<div class="container workshops" id="main-section">
+  <div class="row">
+    <div class="col-md-10 col-md-offset-1 col-sm-12">
+      <div class="row content-section separated">
+        <div class="col-sm-12">
+          <p>We offer a variety of Workshops to help teams advance their skills in many frontend and backend topics. <strong>All of the Workshops can be customized to meet your team's specific needs</strong> - <a href="/contact/">contact us</a> to discuss how we can help you best.</p>
+          <p><strong>All of simplabs's Workshops are taught be <a href="http://twitter.com/michaellnorth?lang=de">Mike North</a>.</strong> Mike is a global speaker, trainer, Front End Masters instructor and Pluralsight author. He <strong>has taught developers all over the world how to use Phoenix, Elixir and Ember.js</strong>.</p>
```
Comment (Turbo87) [inline workshops/index.html]: should this be "simplabs' Workshops"?

**Comment by Turbo87** [comment]
I've only reviewed the resulting pages and aside from the comment above I'm 👍 on this

File: workshops/index.html (hunk 3 above)
Comment (marcoow) [reply workshops/index.html]: yes ;)

Comment (marcoow) [reply workshops/index.html]: fixed

**Review by Turbo87 — APPROVED** [review]

**Review by ganeshbsub — APPROVED** [review]

**Review by gr0uch — APPROVED** [review]
Looks good overall. Is it possible to not commit generated files into source control though?

**Comment by marcoow** [comment]
In this case we have to do that. We can regenerate with the script though.

File: workshops/index.html [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+  </div>
+</div>
+
+<div class="container workshops" id="main-section">
+  <div class="row">
+    <div class="col-md-10 col-md-offset-1 col-sm-12">
+      <div class="row content-section separated">
+        <div class="col-sm-12">
+          <p>We offer a variety of Workshops to help teams advance their skills in many frontend and backend topics. <strong>All of the Workshops can be customized to meet your team's specific needs</strong> - <a href="/contact/">contact us</a> to discuss how we can help you best.</p>
+          <p><strong>All of simplabs' Workshops are taught be <a href="http://twitter.com/michaellnorth?lang=de">Mike North</a>.</strong> Mike is a global speaker, trainer, Front End Masters instructor and Pluralsight author. He <strong>has taught developers all over the world how to use Phoenix, Elixir and Ember.js</strong>.</p>
```
Comment (mike-north) [inline workshops/index.html]: are taught *by*

Comment (marcoow) [reply workshops/index.html]: 👍 fixed

**Pull request merged**